src/api/migrations/20190701162027-create-bookmark.js

```diff
@@ -0,0 +1,44 @@
+export default {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('bookmarks', {
+      slug: {
+        type: Sequelize.STRING,
+        references: {
+          model: 'Articles',
+          key: 'slug'
+        },
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+        allowNull: false
+      },
+      username: {
```

Comment:
Same here @truestbyheart 
something like `authorUsername` makes more sense. (if there is an `author idea` in this feature)

Reply:
The reader is the one that's bookmarking the page

Reply:
I think @nkpremices  is right something that shows that  the user is bookmarking can make it the code more understandable